module/zcommon/zfs_prop.c

```diff
@@ -643,7 +643,12 @@ zfs_prop_init(void)
 	    ZFS_TYPE_VOLUME, "512 to 128k, power of 2",	"VOLBLOCK", B_FALSE,
 	    sfeatures);
 	zprop_register_index(ZFS_PROP_VOLTHREADING, "volthreading",
-	    1, PROP_DEFAULT, ZFS_TYPE_VOLUME, "on | off", "zvol threading",
+#ifdef __FreeBSD__
+	    0,
+#else
+	    1,
```

Comment:
This controls zvol creation time, so pool imported from other OS will have not the intended one.

Reply:
Revert it. The zv_threading does not used on FreeBSD side for now.